From 2231e353571424f6412b988e60572f62929dcccc Mon Sep 17 00:00:00 2001
From: "Dr. David Alan Gilbert" <dgilbert@redhat.com>
Date: Thu, 2 Jun 2016 15:11:48 +0200
Subject: [PATCH 22/24] fw_cfg for 7.2 compatibility

RH-Author: Dr. David Alan Gilbert <dgilbert@redhat.com>
Message-id: <1464880310-29953-5-git-send-email-dgilbert@redhat.com>
Patchwork-id: 70543
O-Subject: [RHEL-7.3 qemu-kvm-rhev PATCH 4/6] fw_cfg for 7.2 compatibility
Bugzilla: 1342015
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>
RH-Acked-by: Bandan Das <bsd@redhat.com>
RH-Acked-by: Eduardo Habkost <ehabkost@redhat.com>

From: "Dr. David Alan Gilbert" <dgilbert@redhat.com>

Laszlo's fw_cfg fix from e6915b5f3a874a467a9a65f7ec1d6ef8d251a51a needs
to be on in the 7.2-compat type.

Signed-off-by: Dr. David Alan Gilbert <dgilbert@redhat.com>
Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 include/hw/compat.h | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/include/hw/compat.h b/include/hw/compat.h
index 5a075e4..45e7419 100644
--- a/include/hw/compat.h
+++ b/include/hw/compat.h
@@ -180,5 +180,13 @@
             .driver   = "virtio-pci",\
             .property = "migrate-extra",\
             .value    = "off",\
+        },{ /* HW_COMPAT_RHEL7_2 */ \
+            .driver   = "fw_cfg_mem",\
+            .property = "dma_enabled",\
+            .value    = "off",\
+        },{ /* HW_COMPAT_RHEL7_2 */ \
+            .driver   = "fw_cfg_io",\
+            .property = "dma_enabled",\
+            .value    = "off",\
         },
 #endif /* HW_COMPAT_H */
-- 
1.8.3.1

